cmd/minishift/cmd/root.go

```diff
@@ -46,7 +48,7 @@ import (
 )
 
 var dirs = [...]string{
```

Comment:
I had also thought about in the same direction but wanted to make it work first. Will fix it.